src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -68,7 +68,7 @@ impl KvmVcpu {
     ///
     /// # Arguments
     ///
-    /// * `id` - Represents the CPU number between [0, max vcpus).
+    /// * `index` - Represents the CPU number between [0, max vcpus].
```

Comment:
Would you mind removing the description for the `fd` argument on line 93, as well? It seems like arguments description for `configure()` function are out of date, too.

Reply:
It should be `[0, max_vcpus)` or if you find that notation confusing, then use `0 <= index < max_vcpus`.

`index` will be strictly smaller than `max_vcpus`.